src/composables/director-composable.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Composable that provides some utilities for director data.
+ */
+export const DirectorComposable = () => {
+  /**
+   * Sorter function based on ES6 map reduce to sort the list of directors in the json.
+   * @param fields the list of fields based on which director list needs to be sorted
+   * @returns the sort function
+   */
+  const fieldSorter = (fields: Array<string>) => {
+    return (a, b) => fields.map(o => {
+      return a['officer'][o] > b['officer'][o] ? 1 : a['officer'][o] < b['officer'][o] ? -1 : 0
+    }).reduce((p, n) => p || n, 0)
+  }
+  return {
+    fieldSorter
+  }
+}
```

Comment:
Do you think this is better as a composable or would it be better as a standalone util function like these?

https://github.com/bcgov/business-edit-ui/tree/main/src/utils

Reply:
I am not sure. It very well might be. Right now what I have converted over is the composition api with a structure of an options api app. Which I don't think uses the composition api to its full potential